cmd/earthly/build_cmd.go

```diff
@@ -558,6 +567,44 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 	return nil
 }
 
+func (app *earthlyApp) getOrgForAnalytics() string {
+	if app.analyticsMetadata.orgName != "" {
+		return app.analyticsMetadata.orgName
+	} else {
+		orgMu.Lock()
+		defer orgMu.Unlock()
+		if app.analyticsMetadata.orgName != "" {
+			return app.analyticsMetadata.orgName
+		}
+	}
```

Comment:
There's a return in the `if`, so the `else` is unnecessary. Also from [Effective Go](https://go.dev/doc/effective_go#if):

> when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

```suggestion
	}
	orgMu.Lock()
	defer orgMu.Unlock()
	if app.analyticsMetadata.orgName != "" {
		return app.analyticsMetadata.orgName
	}
```